lib/vagrant-openshift/action/generate_template.rb

```diff
@@ -40,7 +40,7 @@ def call(env)
           box_info = box_info_data[os][stage]
           box_info[:instance_name] = @options[:name].nil? ? 'openshift_origin_' + inst_ts : @options[:name]
           box_info[:os] = os
-          box_info[:vagrant_guest] = [:centos7, :rhel7].include?(os) ? :redhat : os
+          box_info[:vagrant_guest] = [:centos7, :rhel7, :atomic].include?(os) ? :redhat : os
```

Comment:
Sure I can make an adjustment to call this out as rhel-atomic and then the fedora-atomic can be added later